modules/nw-multus-ipam-object.adoc

```diff
@@ -310,4 +310,47 @@ pod/whereabouts-reconciler-p4sxw 1/1 Running 0 6s
 pod/whereabouts-reconciler-rvfdv 1/1 Running 0 6s
 pod/whereabouts-reconciler-svzw9 1/1 Running 0 6s
 daemonset.apps/whereabouts-reconciler 6 6 6 6 6 kubernetes.io/os=linux 6s
-----
\ No newline at end of file
+----
+
+[id="nw-multus-creating-whereabouts-dualstack-config_{context}"]
+== Creating a DualStack config
+
+DualStack can be configured with `ipRanges`.
```

Comment:
suggestion: 

> Dual stack IP address assignment (for both IPv4 and IPv6 addresses), or multiple IP address assignment, can be configured with the `ipRanges` parameter.

I'm trying to get across that dual stack is the primary use case, but, you can configure n-number of addresses, and I want folks to know that "multiple IP address assignment" works too.